lib/pubsub/subscription.js

```diff
@@ -280,16 +277,6 @@ function Subscription(pubsub, options) {
   this.messageListeners = 0;
   this.paused = false;
 
-  if (is.number(options.timeout)) {
-    this.timeout = options.timeout;
-  } else {
-    // The default timeout used in gcloud-node is 60s, but a pull request times
-    // out around 90 seconds. Allow an extra couple of seconds to give the API a
-    // chance to respond on its own before terminating the connection.
-    var PUBSUB_API_TIMEOUT = 90000;
-    this.timeout = PUBSUB_API_TIMEOUT + 2000;
```

Comment:
Makes sense, I was looking to see if protobuf.js had a timeout option (I believe grpc still uses this under the hood?) and I didn't see anything in their documentation.